src/components/download/DownloadDialog.js

```diff
@@ -78,29 +64,34 @@ export class DownloadDialog extends Component {
         const isSupported = downloadSupport() && !this.state.error;
 
         return (
-            <Dialog open={showDialog} onClose={this.onClose}>
-                <DialogTitle disableTypography={true}>
-                    {i18n.t('Download map')}
-                </DialogTitle>
-                <DialogContent className={classes.content}>
+            <Modal
+                open={showDialog}
+                position="middle"
+                small
+                onClose={this.onClose}
+            >
+                <ModalTitle>{i18n.t('Download map')}</ModalTitle>
+                <ModalContent className={styles.modalContent}>
```

Comment:
I'm unsure about this way of customising styles.
See: https://ui.dhis2.nu/#/advanced-usage

Reply:
Fixed by adding class names to a div rather than the UI element.